cmake/compiler/clang.cmake

```diff
@@ -33,5 +33,20 @@ foreach(isystem_include_dir ${NOSTDINC})
   list(APPEND isystem_include_flags -isystem ${isystem_include_dir})
 endforeach()
 
+execute_process(
+  COMMAND ${CMAKE_C_COMPILER} ${TOOLCHAIN_C_FLAGS} --print-libgcc-file-name
+  OUTPUT_VARIABLE LIBGCC_FILE_NAME
+  OUTPUT_STRIP_TRAILING_WHITESPACE
+  )
+
+assert_exists(LIBGCC_FILE_NAME)
+
+get_filename_component(LIBGCC_DIR ${LIBGCC_FILE_NAME} DIRECTORY)
+
+assert_exists(LIBGCC_DIR)
+
+list(APPEND LIB_INCLUDE_DIR "-L\"${LIBGCC_DIR}\"")
```

Comment:
"-L" is not portable. Can't we use link_directories() instead?

Reply:
AFAICT "-L" is portable across clang compilers.

https://clang.llvm.org/docs/ClangCommandLineReference.html#linker-flags

Reply:
Sorry my mind forgot the context :)